hackathon_site/event/urls.py

```diff
@@ -1,6 +1,8 @@
 from django.contrib.auth import views as auth_views
 from django.urls import path
 from event.views import IndexView, DashboardView
+from django.urls import reverse_lazy
```

Comment:
You can combine this with the import from django.urls in line 2. I usually put all library imports first (in alphabetical order), then all app imports below.